src/ext/oblivc/obliv_bits.c

```diff
@@ -26,6 +26,15 @@ static __thread ProtocolDesc *currentProto;
 static inline bool known(const OblivBit* o) { return !o->unknown; }
 
 // --------------------------- Transports -----------------------------------
+
+static bool transportProfilingEnabled = false;
```

Comment:
The reasoning behind using a global switch was to avoid the machinery associated with propagating profiled-ness through split protocols and to give users a protocol-agnostic way to enable it, assuming we have other protocols in the future. Point taken, though. I will look at making the relevant adjustments. Ironically, I originally wrote this with a protocolAcceptProfiledTcp2() function, but changed it.